backend_test.go

```diff
@@ -347,8 +347,18 @@ func testDBBatch(t *testing.T, backend BackendType) {
 	require.NoError(t, batch.Set([]byte("c"), []byte{3}))
 	assertKeyValues(t, db, map[string][]byte{})
 
+	batchSize, err := batch.GetByteSize()
+	require.NoError(t, err)
+	// batchSize should be grater than 6 which is the total size of keys and values
+	require.GreaterOrEqual(t, batchSize, uint64(6))
```

Comment:
Why `GreaterOrEqual` and not `Equal`?

Reply:
because for goleveldb, pebbledb, rocksdb, perfixdb, we get the size of the actual LSM batch's data, instead of the total size of keys and values.

Reply:
I see. Please add this information as a godoc for the batch interface method. Something like "Depending on the implementation, this may return the size of the underlying LSM batch, including the size of additional metadata on top of the expected key and value total byte count."

Then, we should also document the behavior on top of every concrete implementation.